code/modules/research/rdconsole.dm

```diff
@@ -1090,9 +1085,9 @@ proc/CallMaterialName(ID)
 					temp_materials += " | "
 					if (!linked_imprinter.check_mat(D, M))
 						check_materials = 0
-						temp_materials += " <span class='bad'>[D.materials[M]/coeff] [CallMaterialName(M)]</span>"
+						temp_materials += " <span class='bad'>[D.materials[M]] [CallMaterialName(M)]</span>"
 					else
-						temp_materials += " [D.materials[M]/coeff] [CallMaterialName(M)]"
+						temp_materials += " [D.materials[M]] [CallMaterialName(M)]"
 				if (check_materials)

```

Comment:
You'll probably need to split the reagent and material displays into separate categories. Is the acid REALLY stored in the `materials` part of the design for the imprinter, while the protolathe has reagents and materials separate?